### PR TITLE
Basic layout

### PR DESCRIPTION
**Pull request by sahil505**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "moment": "^2.18.1",
+    "postcss-discard-comments": "^2.0.4",
+    "postcss-pxtorem": "^4.0.1",
+    "postcss-rtl": "^0.5.10",
+    "postcss-sorting": "^2.1.0",
+    "pug": "^2.0.0-rc.2",
+    "stylelint": "^7.10.1",
+    "stylelint-order": "^0.4.4",
+    "stylelint-scss": "^1.4.4",
+    "xo": "^0.18.2"
```
Comment (hanakin) [inline package.json]: what version of Js are you using?

File: views/template/demo.pug [hunk 2]
```diff
@@ -0,0 +1,8 @@
+extends layout.pug
+
+block head
+	center
```
Comment (hanakin) [inline views/template/demo.pug]: do not use center tags. You are also missing the navigation on this page. You need a navigation block here not a head block?

File: views/template/documentation.pug [hunk 3]
```diff
@@ -0,0 +1,8 @@
+extends layout.pug
+
+block head
+	center
```
Comment (hanakin) [inline views/template/documentation.pug]: do not use center tags. You are also missing the navigation on this page. You need a navigation block here not a head block?

File: views/template/index.pug [hunk 4]
```diff
@@ -0,0 +1,27 @@
+extends layout.pug
+
+block head
+	.container
+		.logo
+			img.head-logo(src="/assets/img/phpbblogo.png")
```
Comment (hanakin) [inline views/template/index.pug]: not necessary remove this

File: views/template/index.pug [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+			h1 Styleguide
+			img.icon(src="/assets/img/github_icon.png" alt="github")
```
Comment (hanakin) [inline views/template/index.pug]: you need to be using base-l also better to use svg imgs.

File: views/template/index.pug [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+		.content
+			p A Styleguide Driven Development System for creating, managaing, customizing, & editing themes for phpBB.
+			p GSoC 2017 project being developed by Sahil Khokhar.
+		.home_btn
```
Comment (hanakin) [inline views/template/index.pug]: we do not use _ per base-l in class names also the naming convention is wrong as this is a wrapping div? Pls use component development approaches in structuring your html.

File: views/template/index.pug [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+			- var btnType = "info"
+			- var btnSize = "lg"
```
Comment (hanakin) [inline views/template/index.pug]: what is all of this? not needed?

File: views/template/index.pug [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+			a(href="./demo")
+				button(type="button" class="btn btn-${btnType} btn-${btnSize}") Demo
```
Comment (hanakin) [inline views/template/index.pug]: do not need the button if you are using an a tag. also again base-l

File: views/template/index.pug [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+			|
+			|
+			a(href="./documentation")
+				button(type="button" class="btn btn-${btnType} btn-${btnSize}") Documentation
```
Comment (hanakin) [inline views/template/index.pug]: do not need the button if you are using an a tag. also again base-l

File: views/template/index.pug [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+			|
+			|
+			a(href="./settings")
+				button(type="button" class="btn btn-${btnType} btn-${btnSize}") App Settings
```
Comment (hanakin) [inline views/template/index.pug]: do not need the button if you are using an a tag. also again base-l

File: views/template/settings.pug [hunk 11]
```diff
@@ -0,0 +1,8 @@
+extends layout.pug
+
+block head
+	center
```
Comment (hanakin) [inline views/template/settings.pug]: do not use center tags. You are also missing the navigation on this page. You need a navigation block here not a head block?

**Review by hanakin — CHANGES_REQUESTED** [review]

File: app.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
+var express = require('express');
```
Comment (hanakin) [inline app.js]: this file has linting errors via xo...as you are not using ecma 2015 but the version of xo you are using is for ecma 2015.

File: assets/css/style.css [hunk 13]
```diff
@@ -0,0 +1,39 @@
+.container {
```
Comment (hanakin) [inline assets/css/style.css]: please setup your workflow (gulpjs) and use base-l for all your css

File: views/template/index.pug (hunk 4 above)
Comment (sahil505) [reply views/template/index.pug]: I just kept it as a placeholder for the SDD logo. Do you want to keep this blank for now?

File: views/template/demo.pug (hunk 2 above)
Comment (sahil505) [reply views/template/demo.pug]: Won't use center tags.
Sidebar right? (Although the block can be called navigation)
Yes, I'm aware of that, just wanted to set up the pages first. Will add it in the next push.

File: views/template/documentation.pug (hunk 3 above)
Comment (sahil505) [reply views/template/documentation.pug]: Won't use center tags.
Top navbar right? (Although the block can be called navigation)
Yes, I'm aware of that, just wanted to set up the pages first. Will add it in the next push.

File: package.json (hunk 1 above)
Comment (sahil505) [reply package.json]: Changed the xo version to 0.16.0

File: views/template/index.pug (hunk 7 above)
Comment (sahil505) [reply views/template/index.pug]: These are the attributes that we can use inside a button tag. I've removed it for now.

File: views/template/index.pug (hunk 8 above)
Comment (sahil505) [reply views/template/index.pug]: Okay, so just CSS around that link? 
Yeah, will shift to base-l in my next push.

File: views/template/index.pug (hunk 9 above)
Comment (sahil505) [reply views/template/index.pug]: Yeah, will shift to base-l in my next push.

File: views/template/index.pug (hunk 10 above)
Comment (sahil505) [reply views/template/index.pug]: Yeah, will shift to base-l in my next push.

File: views/template/settings.pug (hunk 11 above)
Comment (sahil505) [reply views/template/settings.pug]: Won't use center tags.
Top navbar right? (Although the block can be called navigation)
Yes, I'm aware of that, just wanted to set up the pages first. Will add it in the next push.

File: views/template/index.pug (hunk 4 above)
Comment (hanakin) [reply views/template/index.pug]: yes

File: views/template/index.pug (hunk 5 above)
Comment (sahil505) [reply views/template/index.pug]: okay, any specific reason to with SVG images?

File: app.js (hunk 12 above)
Comment (sahil505) [inline app.js]: taken care of

File: assets/css/style.css (hunk 13 above)
Comment (sahil505) [reply assets/css/style.css]: taken care of

File: views/demo.pug [hunk 14]
```diff
@@ -0,0 +1,29 @@
+extends layout.pug
+
+block head
+	.navbar
```
Comment (hanakin) [inline views/demo.pug]: This block needs relooked at for proper component design. All HTML elements should have a class as we never style elements directly.

File: views/demo.pug [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+				a(href="/documentation") Documentation
+			li
+				a(href="/settings") App Settings
+		a(href="https://github.com/sahil505/StyleGuideDesigner", target="_blank"): img.icon(src="/img/github-icon-1.svg" alt="github")
+	hr
+block content
+	.container-fluid
+		.row
+			.col-sm-3.col-lg-2
+				nav.side-navbar.navbar-default.navbar-fixed-side
```
Comment (hanakin) [inline views/demo.pug]: Are u using bootstrap? You have too many classes here and are not following proper class naming conventions all that is needed is c-sidebar

File: views/documentation.pug [hunk 16]
```diff
@@ -0,0 +1,18 @@
+extends layout.pug
+
+block head
+	.navbar
```
Comment (hanakin) [inline views/documentation.pug]: Your approach has a lot of redundant code maybe consider using a partial or better yet make this the default and override in index

File: assets/css/style.css (hunk 13 above)
Comment (hanakin) [reply assets/css/style.css]: This all needs redone properly and in the theme file

**Review by hanakin — CHANGES_REQUESTED** [review]

File: package.json [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+    "gulp-autoprefixer": "^4.0.0",
+    "gulp-cssnano": "^2.1.2",
+    "gulp-data": "^1.2.1",
+    "gulp-nunjucks-render": "^2.2.1",
+    "gulp-postcss": "^7.0.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-sass": "^3.1.0",
+    "gulp-sourcemaps": "^2.6.0",
+    "gulp-stylefmt": "^1.1.0",
+    "moment": "^2.18.1",
```
Comment (hanakin) [inline package.json]: no needed

File: package.json [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+    "postcss-discard-comments": "^2.0.4",
```
Comment (hanakin) [inline package.json]: not needed

**Review by hanakin — CHANGES_REQUESTED** [review]

File: views/documentation.pug (hunk 16 above)
Comment (sahil505) [reply views/documentation.pug]: can I make it a separate file and include it in demo/doc/settings? or override will also be fine I guess, which one do you want to go with?

File: views/demo.pug (hunk 15 above)
Comment (sahil505) [reply views/demo.pug]: I'm using classes names similar to bootstrap but not exactly bootstrap. I'll simplify the code.

File: package.json (hunk 18 above)
Comment (sahil505) [reply package.json]: So remove the lines 42-46?

File: views/demo.pug (hunk 14 above)
Comment (sahil505) [reply views/demo.pug]: Okay, I'll edit it.

File: package.json (hunk 18 above)
Comment (hanakin) [reply package.json]: Remove moment and discard-comments lines

File: views/documentation.pug (hunk 16 above)
Comment (hanakin) [reply views/documentation.pug]: Both? Technically the nav it self should be a partial so u can include it on the index. And just have the navbar as default and include the nav in the default navbar

File: views/demo.pug (hunk 15 above)
Comment (hanakin) [reply views/demo.pug]: yes do not use bootstrap anything pls, follow the base-l guidlines

File: package.json (hunk 18 above)
Comment (sahil505) [reply package.json]: done

File: package.json (hunk 17 above)
Comment (sahil505) [reply package.json]: done

File: views/documentation.pug (hunk 16 above)
Comment (sahil505) [reply views/documentation.pug]: done

File: views/demo.pug [hunk 19]
```diff
@@ -4,13 +4,13 @@ block content
...
 					ul
-						li
-							a(href="#") Link-1
-						li
-							a(href="#") Link-2
-						li
-							a(href="#") Link-3
+						a(href="#")
+							li Link-1
+						a(href="#")
```
Comment (hanakin) [inline views/demo.pug]: This is now backwards why are you wrapping li with a tags? Also you still are missing the point that you have to give all of these classes in component design. It's illegal to style HTML tags.

**Review by hanakin — CHANGES_REQUESTED** [review]

File: views/demo.pug (hunk 19 above)
Comment (hanakin) [reply views/demo.pug]: I will build the sidebar for you properly as an exp to help you better understand when I get a chance

Comment (hanakin) [reply views/demo.pug]: I will build the sidebar for you properly as an exp to help you better understand when I get a chance

Comment (sahil505) [reply views/demo.pug]: I was wrapping  `<li>` inside `<a>` because it provides an ease of usability i:e a user can also click nearby the text to access the link which is maybe easy to use than just clicking on the text to access the link. 
And yes, I'll make classes in my next commit.

Comment (sahil505) [reply views/demo.pug]: I've corrected this.

Comment (hanakin) [reply views/demo.pug]: this is invalid html you can not wrap a block level element with a non block level element. If you want the whole thing to be the link you style the a to act like a block and define all your layout on the a. you actually should never have an styling on li. You always want to style the a

File: views/layout.pug [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+	body
+		block head
+			.navbar
+				ul
```
Comment (hanakin) [inline views/layout.pug]: needs a class of navbar-nav

File: views/layout.pug [hunk 21]
```diff
@@ -0,0 +1,29 @@
...
+					li
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-item

File: views/layout.pug [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+						a.sdd-logo(href="/") Logo
```
Comment (hanakin) [inline views/layout.pug]: why is this here? should be its own div

File: views/layout.pug [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+					li
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-item

File: views/layout.pug [hunk 24]
```diff
@@ -0,0 +1,29 @@
...
+						a(href="/demo") Demo
+					li
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-item

File: views/layout.pug [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+						a(href="/documentation") Documentation
+					li
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-item

File: views/layout.pug [hunk 26]
```diff
@@ -0,0 +1,29 @@
...
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+	body
+		block head
+			.navbar
+				ul
+					li
+						a.sdd-logo(href="/") Logo
+					li
+						a(href="/demo") Demo
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-link

File: views/layout.pug [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+					li
+						a(href="/documentation") Documentation
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-link

File: views/layout.pug [hunk 28]
```diff
@@ -0,0 +1,29 @@
...
+					li
+						a(href="/settings") App Settings
```
Comment (hanakin) [inline views/layout.pug]: needs class of navbar-link

File: views/layout.pug [hunk 29]
```diff
@@ -0,0 +1,29 @@
...
+				a(href="https://github.com/sahil505/StyleGuideDesigner", target="_blank"): img.icon(src="/img/github-icon-1.svg" alt="github")
```
Comment (hanakin) [inline views/layout.pug]: the anchor needs a class and this needs to be a component

File: views/layout.pug [hunk 30]
```diff
@@ -0,0 +1,29 @@
...
+			hr
+		block content
+		block foot
+			footer
```
Comment (hanakin) [inline views/layout.pug]: needs a class of footer

File: views/demo.pug [hunk 31]
```diff
@@ -0,0 +1,16 @@
+extends layout.pug
+
+block content
+	.container-fluid
```
Comment (hanakin) [inline views/demo.pug]: sidebar should not bee in a container its supposed to be outside of the container. also no grid is necessary for a sidebar

File: views/index.pug [hunk 32]
```diff
@@ -0,0 +1,22 @@
+extends layout.pug
+
+block head
+	header
+		.logo
+			h1 <br>Styleguide
```
Comment (hanakin) [inline views/index.pug]: remove the br and just make the h1 the .logo you do not need the extra div

File: .travis.yml [hunk 33]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
+  - "6.1"
+  - "5.11"
+  - "6.9.4"
```
Comment (hanakin) [inline .travis.yml]: you only need 6, 7 this will test all sub versions

Comment (sahil505) [reply .travis.yml]: okay

File: src/scss/components/_navbar.scss [hunk 34]
```diff
@@ -0,0 +1,59 @@
...
+
+	&-menu {
+		list-style-type: none;
+		margin: 0;
+		padding: 0;
+	}
+
+	&-menu-link {
+		float: left;
+		font-size: 1rem;
```
Comment (hanakin) [inline src/scss/components/_navbar.scss]: rem are automatically built in the render process just use px

File: views/demo.pug (hunk 31 above)
Comment (sahil505) [reply views/demo.pug]: yeah, i still have to clean up the sidebar code and all the classes and all. On it.

File: views/layout.pug [hunk 35]
```diff
@@ -0,0 +1,31 @@
...
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/style.css")
+	body
+		block head
+			.c-navbar
+				ul.c-navbar-menu
+					li.c-navbar-menu-item
+						a.c-navbar-menu-logo(href="/") Logo
+					li
```
Comment (hanakin) [inline views/layout.pug]: you forgot all the other li classes

File: views/layout.pug [hunk 36]
```diff
@@ -0,0 +1,31 @@
...
+						a.c-navbar-menu-link(href="/documentation") Documentation
+					li
+						a.c-navbar-menu-link(href="/settings") App Settings
+					li
+						a.c-navbar-menu-icon(href="#") Git
+			hr
+		block content
+		block foot
+			footer
```
Comment (hanakin) [inline views/layout.pug]: should be a class of .footer here

File: views/layout.pug [hunk 37]
```diff
@@ -0,0 +1,31 @@
...
+				hr
+				a(href="https://github.com/sahil505/StyleGuideDesigner", target="_blank") Git
```
Comment (hanakin) [inline views/layout.pug]: this needs an icon object class

File: views/index.pug (hunk 32 above)
Comment (sahil505) [reply views/index.pug]: okay

File: views/layout.pug [hunk 38]
```diff
@@ -0,0 +1,31 @@
...
+						a.c-navbar-menu-link(href="/documentation") Documentation
+					li
+						a.c-navbar-menu-link(href="/settings") App Settings
+					li
+						a.c-navbar-menu-icon(href="#") Git
+			hr
+		block content
+		block foot
+			footer
+				hr
```
Comment (hanakin) [inline views/layout.pug]: do not use hrs just use borders on the block level elements

**Review by hanakin — CHANGES_REQUESTED** [review]

File: views/layout.pug [hunk 39]
```diff
@@ -0,0 +1,31 @@
+doctype html
+
+html
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/style.css")
```
Comment (hanakin) [inline views/layout.pug]: should be theme.css

File: views/index.pug [hunk 40]
```diff
@@ -0,0 +1,22 @@
...
+			p GSoC 2017 project being developed by Sahil Khokhar.
+		.home-navigation
+			a.button(href="./demo") Demo
+			|
+			|
+			a.button(href="./documentation") Documentation
+			|
+			|
+			a.button(href="./settings") App Settings
+		hr
```
Comment (hanakin) [inline views/index.pug]: remove this

File: views/index.pug [hunk 41]
```diff
@@ -0,0 +1,22 @@
+extends layout.pug
+
+block head
+	header
+		.logo
+			h1 <br>Styleguide
+			h4 Git
```
Comment (hanakin) [inline views/index.pug]: this should be a span not an h4 inside of the h1 above

File: views/index.pug [hunk 42]
```diff
@@ -0,0 +1,22 @@
+extends layout.pug
+
+block head
+	header
```
Comment (hanakin) [inline views/index.pug]: needs a class of header

File: gulpfile.js [hunk 43]
```diff
@@ -0,0 +1,107 @@
...
+var sass = require('gulp-sass');
+var rename = require('gulp-rename');
+var sourcemaps = require('gulp-sourcemaps');
+var cssnano = require('gulp-cssnano');
+var postcss = require('gulp-postcss');
+var stylefmt = require('gulp-stylefmt');
+var sorting = require('postcss-sorting');
+var torem = require('postcss-pxtorem');
+var sortOrder = require('./.postcss-sorting.json');
+var pkg = require('./package.json');
```
Comment (hanakin) [inline gulpfile.js]: do not need this

File: gulpfile.js [hunk 44]
```diff
@@ -0,0 +1,107 @@
...
+					'padding-bottom'],
+				selectorBlackList: [],
+				replace: true,
+				mediaQuery: false,
+				minPixelValue: 0
+			})
+		])
+	)
+	.pipe(stylefmt())
+	.pipe(rename({
```
Comment (hanakin) [inline gulpfile.js]: do not need to rename here remove this block

File: gulpfile.js [hunk 45]
```diff
@@ -0,0 +1,107 @@
...
+	return css;
+});
+
+gulp.task('clean', function () {
+	del(['dist']);
+});
+
+gulp.task('minify', ['css'], function () {
+	var css = gulp
+	.src(build.css + '/*.' + pkg.version + '.css')
```
Comment (hanakin) [inline gulpfile.js]: should be .src(build.css + '/theme.css')

File: assets/css/style.css [hunk 46]
```diff
@@ -0,0 +1,157 @@
+footer,
```
Comment (hanakin) [inline assets/css/style.css]: this entire file needs removed

File: assets/css/style.css [hunk 47]
```diff
@@ -0,0 +1,157 @@
...
+header {
+	text-align: center;
+}
+
+.container {
+	text-align: center;
+}
+
+.c-navbar {
```
Comment (hanakin) [inline assets/css/style.css]: this is a component pls code in scss component file?

File: assets/css/style.css [hunk 48]
```diff
@@ -0,0 +1,157 @@
...
+.c-navbar-menu-icon {
+	float: right;
+	font-size: 1.0rem;
+	color: #868686;
+	text-align: center;
+	padding: 17px 16px;
+	text-decoration: none;
+}
+
+.container-fluid {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 49]
```diff
@@ -0,0 +1,157 @@
...
+	margin-right: auto;
+	margin-left: auto;
+}
+
+.container-fluid:before,
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 50]
```diff
@@ -0,0 +1,157 @@
...
+.row:before,
+.row:after,
+.container-fluid:after {
+	display: table;
+	content: " ";
+}
+
+.container-fluid:after,
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 51]
```diff
@@ -0,0 +1,157 @@
...
+.row:after {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 52]
```diff
@@ -0,0 +1,157 @@
...
+	clear: both;
+}
+
+.row {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 53]
```diff
@@ -0,0 +1,157 @@
...
+	margin-right: -15px;
+	margin-left: -15px;
+}
+
+.col-sm-3,
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 54]
```diff
@@ -0,0 +1,157 @@
...
+.col-sm-3, 
+.col-lg-2, 
+.col-sm-9 {
+	position: relative;
+	min-height: 1px;
+	padding-right: 15px;
+	padding-left: 15px;
+}
+
+@media (min-width: 768px) {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 55]
```diff
@@ -0,0 +1,157 @@
...
+	.col-sm-9 {
+		float: left;
+		width: 75%;
+	}
+}
+
+@media (min-width: 768px) {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 56]
```diff
@@ -0,0 +1,157 @@
...
+	.col-sm-3 {
+		float: left;
+		width: 25%;
+	}
+}
+
+@media (min-width: 1200px) {
```
Comment (hanakin) [inline assets/css/style.css]: not needed

File: assets/css/style.css [hunk 57]
```diff
@@ -0,0 +1,157 @@
...
+	.col-lg-2{
+		float: left;
+		width: 16.66666667%;
+	}
+}
+
+.c-sidebar {
```
Comment (hanakin) [inline assets/css/style.css]: needs to be in its own file

File: assets/css/style.css [hunk 58]
```diff
@@ -0,0 +1,157 @@
...
+	cursor: pointer;
+}
+
+.c-sidemenu-link:hover {
+	text-decoration: none;
+	background-color: #eeeeee;
+	color: #212121;
+}
+
+@media (min-width: 768px) {
```
Comment (hanakin) [inline assets/css/style.css]: not sure what this is? not needed

File: views/index.pug (hunk 41 above)
Comment (sahil505) [reply views/index.pug]: h4 is not inside of h1, I've not given any indentation. anyways this is a temp for the git icon

File: assets/css/theme.0.12.0.css [hunk 59]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (hanakin) [inline assets/css/theme.0.12.0.css]: you can delete all these theme.0.12.0 files

File: src/scss/components/_sidebar.scss [hunk 60]
```diff
@@ -0,0 +1,66 @@
...
+
+
+//
+// @scss
+.c-sidebar {
+	position: relative;
+	padding-right: 15px;
+	padding-left: 15px;
+	float: left;
+	width: 16.66666667%;
```
Comment (hanakin) [inline src/scss/components/_sidebar.scss]: this should not be a percentage, use px

File: src/scss/components/_sidebar.scss [hunk 61]
```diff
@@ -0,0 +1,66 @@
...
+	text-align: center;
+	min-height: 500px;
+	margin: 0;
+	border: 1px solid transparent;
+	background-color: #f1f1f1;
+	border-color: #e7e7e7;
+	border-radius: 4px;
```
Comment (hanakin) [inline src/scss/components/_sidebar.scss]: why is there are border-radius?

File: views/demo.pug [hunk 62]
```diff
@@ -0,0 +1,29 @@
+extends layout.pug
+
+block content
+	.container
```
Comment (hanakin) [inline views/demo.pug]: remove the container

File: views/index.pug [hunk 63]
```diff
@@ -0,0 +1,20 @@
+extends layout.pug
+
+block head
+	header.c-header
+		.c-header-logo
+			h1 <br>Styleguide
```
Comment (hanakin) [inline views/index.pug]: again this is all wrong

the h1 should be the logo class remove the extra div and remove the br. also the h3 below should by in the h1 and a span

**Review by hanakin — APPROVED** [review]
